src/site/content/en/blog/rakuten/index.md

```diff
@@ -0,0 +1,166 @@
+  <figcaption>
+    The impact of LCP on conversion rate.
+  </figcaption>
+</figure>
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/MBCTKocekiENQSQBS6wO.jpg", alt="LCP bucketed by revenue per visitor and LCP time. Users with lower LCP provided more revenue, with 26.09% more revenue provided per user when LCP was one second or lower.", width="800", height="376" %}
+  <figcaption>
+    The impact of LCP on revenue per visitor.
+  </figcaption>
+</figure>
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/mFfKxOV5snRw2e6lEqC5.jpg", alt="LCP bucketed by average order value and LCP time. Users with lower LCP had a 11.26% higher average order value when LCP was one second or lower.", width="800", height="376" %}
+  <figcaption>
+    The impact of LCP on average order value.
+  </figcaption>
+</figure>
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/hzfFyMF9UM0PD0y1dq9V.jpg", alt="FID bucketed by conversion rate and FID time. Users who converted more frequently converted when FID was lower, with 55.88% of users converting with an FID of 50 milliseconds or lower.", width="800", height="362" %}
+  <figcaption>
+    The impact of FID on conversion rate.
+  </figcaption>
+</figure>
+
+#### Monitoring
+
+The team built a performance monitoring dashboard using the data collected in the field and business intelligence tool. This is important for monitoring progress and preventing regressions.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/nd87NaZ2i9cG905ChUdU.jpg", alt="A screenshot of Rakuten 24's internal performance monitoring dashboard for each of the Core Web Vitals (LCP, CLS, and FID).", width="800", height="354" %}
+  <figcaption>
+    Performance monitoring dashboard.
+  </figcaption>
+</figure>
+
+### A/B test
+
+Believing that A/B tests are a good way to measure the business impact of performance optimizations, the team optimized one of their landing pages for Core Web Vitals then compared the optimized version with the original page via A/B test for a month. They picked a landing page with significant traffic and conversion so that the test could achieve meaningful results. During the test duration, 50% of the traffic was sent to the optimized landing page (version A), and 50% was sent to the original page (version B). The only difference between version A and version B was that version A was optimized for Core Web Vitals and there were no other functional or visual differences.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pN5i17Rh7VxrObYlgdiu.jpg", alt="A screenshot of a mobile A/B test for the Rakuten 24 website. Each version was visually and functionally the same, with version A optimized for better Core Web Vitals.", width="800", height="1129" %}
```

Comment:
@malchata Hi, this image looks too big on PC screen so can you replace it with the following?
https://drive.google.com/file/d/1Yoktvq9Oo2P9yeiMWMws_bBntOXST_-A/view?usp=sharing
(I originally meant to make this part into a 2-column content like text-image but if this cannot be two columns, I think an image with background on both sides will work better)